config/v1/types_infrastructure.go

```diff
@@ -580,12 +580,74 @@ const (
 type GCPPlatformSpec struct{}
 
 // GCPPlatformStatus holds the current status of the Google Cloud Platform infrastructure provider.
+// +kubebuilder:validation:XValidation:rule="!has(oldSelf.resourceLabels) && !has(self.resourceLabels) || has(oldSelf.resourceLabels) && has(self.resourceLabels)",message="resourceLabels may only be configured during installation"
+// +kubebuilder:validation:XValidation:rule="!has(oldSelf.resourceTags) && !has(self.resourceTags) || has(oldSelf.resourceTags) && has(self.resourceTags)",message="resourceTags may only be configured during installation"
 type GCPPlatformStatus struct {
+	// OrganizationID is the Organization ID the new GCP resources created for the cluster.
```

Comment:
Should start with the json tag version of the name
```suggestion
	// organizationID is the Organization ID the new GCP resources created for the cluster.
```